keyboards/era/divine/readme.md

```diff
@@ -0,0 +1,25 @@
+# DIVINE
+
+DIVINE Keyboard
```

Comment:
No image? 
It's not necessary, but it's nice to have for users/reference.

Reply:
Currently, only the PCB is completed, so unfortunately there are no keyboard photos yet. :(